lib/ex_ice/priv/ice_agent.ex

```diff
@@ -3085,7 +3085,7 @@ defmodule ExICE.Priv.ICEAgent do
           """)
 
           ice_agent = put_in(ice_agent.local_cands[local_cand.base.id], local_cand)
-          ice_agent = close_candidate(ice_agent, local_cand)
```

Comment:
I guess this line is responsible for the problem described in the previous comment.